mpmath/matrices/matrices.py

```diff
@@ -993,3 +993,26 @@ def mnorm(ctx, A, p=1):
             return max(ctx.fsum((A[i,j] for j in range(n)), absolute=1) for i in range(m))
         else:
             raise NotImplementedError("matrix p-norm for arbitrary p")
+        
+    def kron(ctx, *args):
+        """
+        Calculates the tensor products of 2 or more matrices, similar to numpy.kron
```

Comment:
We should have a proper docstring.
1. It's not the tensor product, but a special form of it.
2. We can't just reference numpy.kron here

See other docstrings in the module.